pkg/controller/setup.go

```diff
@@ -31,11 +31,18 @@ func SetupControllers(mgr ctrl.Manager, runtimes map[string]runtime.Runtime, opt
 	if err := runtimeRec.SetupWithManager(mgr, options); err != nil {
 		return "TrainingRuntime", err
 	}
+	clRuntimeRec := NewClusterTrainingRuntimeReconciler(
+		mgr.GetClient(),
+		mgr.GetEventRecorderFor("trainer-clustertrainingruntime-controller"),
+	)
+	if err := clRuntimeRec.SetupWithManager(mgr, options); err != nil {
+		return "ClusterTrainingRuntime", err
```

Comment:
Shall we use constants for these ?
https://github.com/kubeflow/trainer/blob/fcea32a3b695293f62790ed1d204e3fbd0f67395/pkg/apis/trainer/v1alpha1/trainingruntime_types.go#L30

Reply:
SGTM

Reply:
Done.